script.js

```diff
@@ -0,0 +1,41 @@
+let select = $('#movie-select')
+
+$('#search').submit(e => {
+  e.preventDefault()
+  let input = $('#movie-search')
```

Comment:
If you wanted to use `const` in place of `let` here you could. You're not changing either of these values during a particular invocation of this function. The same goes for variables used similarly later on in your code.